patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -1,14 +1,22 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/EntityRenderer.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/EntityRenderer.java
-@@ -211,6 +211,7 @@
+@@ -67,6 +67,7 @@
+ import net.minecraft.world.GameType;
+ import net.minecraft.world.World;
+ import net.minecraft.world.biome.Biome;
++import net.minecraftforge.common.ForgeHooks;
```

Comment:
Do not use imports in patches, instead use the fully qualified name where it is used.

Reply:
Fixed in latest commit.